extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/StreamReaderOptions.java

```diff
@@ -6,17 +6,7 @@
 import io.deephaven.extensions.barrage.ColumnConversionMode;
 import org.immutables.value.Value;
 
-@Value.Immutable
-@SimpleStyle
 public abstract class StreamReaderOptions {
-    public static StreamReaderOptions of(BarrageSnapshotOptions options) {
-        return ImmutableStreamReaderOptions.of(options.useDeephavenNulls(), options.columnConversionMode());
-    }
-
-    public static StreamReaderOptions of(BarrageSubscriptionOptions options) {
-        return ImmutableStreamReaderOptions.of(options.useDeephavenNulls(), options.columnConversionMode());
-    }
-
     @Value.Parameter
```

Comment:
I think this should be an interface, and you can drop the remaining annotations from immutables.